weaver/core/relay/Cargo.toml

```diff
@@ -34,8 +37,11 @@ futures = { version = "0.3.27" }
 base64 = "0.20.0"
 reqwest = { version = "0.11.16", features = ["json"] }
 serde_json = "1.0.95"
-# cacti_weaver_protos_rs = { path = "./protos-rs" }
-cacti_weaver_protos_rs = "2.0.0-alpha.1"
+# cacti_weaver_protos_rs = "2.0.0-alpha.1"
+cacti_weaver_protos_rs = { path = "../../common/protos-rs/pkg" }
```

Comment:
Please undo this line change, as otherwise when building Docker image for relay we'll have to give whole repository as context directory, which will slow down the build of docker image.

Reply:
@sandeepnRES yeah that makes sense. Done.